test/e2e/funcs/feature.go

```diff
@@ -66,13 +66,18 @@ const DefaultPollInterval = time.Millisecond * 500
 
 type onSuccessHandler func(o k8s.Object)
 
-// AllOf runs the supplied functions in order.
+// AllOf runs the supplied functions in order. If a function fails the test and
+// the environment is configured to fail fast (e2e-framework's -fail-fast flag)
```

Comment:
looks like we are specifying the intent to fail fast with `-test.failfast -fail-fast` in the e2e step https://github.com/crossplane/crossplane/blob/master/.github/workflows/ci.yml#L306C47-L306C72 - were you able to also verify that the flag is making it down into this testing context correctly?

Reply:
yes, in our specific case, we do build our [environment from the flags](https://github.com/crossplane/crossplane/blob/f626e8a530e2ecaecb7cd59633f0871836e4aa1c/test/e2e/main_test.go#L91), whether another user will pass down that correctly it's up to the user.